modules/data.atmosphere/inst/scripts/runtest-download.NOAA_GEFS.R

```diff
@@ -0,0 +1,38 @@
+#This program is to test the R script donload.NOAA.R during development.
+#It accepts a command line argument specifying the case test number to run, otherwise, it
+#defaults to the first case.
+
+test_no = 1
+args = commandArgs(trailingOnly = TRUE)
+if (length(args) > 0) {
+  test_no = as.integer(args[1])
+}
+
+source("~/pecan/modules/data.atmosphere/R/download.NOAA_GEFS.R")
```

Comment:
For the future, instead of this, look at the [`devtools::load_all`](https://www.rdocumentation.org/packages/devtools/versions/1.13.5/topics/load_all) function.